experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -622,6 +631,66 @@ describe('fetch', () => {
     });
   });
 
+  describe('when response has url', () => {
+    beforeEach(async () => {
+      const propagateTraceHeaderCorsUrls = [url];
+      await prepareData(redirectedUrl, { propagateTraceHeaderCorsUrls });
+    });
+
+    afterEach(() => {
+      clearData();
+    });
+
+    it('span should have correct attributes', () => {
+      const span: tracing.ReadableSpan = exportSpy.args[1][0][0];
+      const attributes = span.attributes;
+      const keys = Object.keys(attributes);
+
+      assert.ok(
+        attributes[keys[0]] !== '',
```

Comment:
Is there a reason we are using `keys[n]` to access the keys and not `attributes[prop]`? It makes it a lot harder to figure out what each key is supposed to be.